utils/maven-plugin/pom.xml

```diff
@@ -15,12 +15,22 @@
     <packaging>maven-plugin</packaging>
     <name>apicurio-registry-maven-plugin</name>
 
+    <properties>
+        <resteasy-client.version>4.5.6.Final</resteasy-client.version>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>io.apicurio</groupId>
             <artifactId>apicurio-registry-rest-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client</artifactId>
+            <version>${resteasy-client.version}</version>
+        </dependency>
+
```

Comment:
This is fine for now - but we should revisit this requirement if we update the client to stop using jax-rs classes.  @carlesarnal